Library/Formula/ace.rb

```diff
@@ -32,5 +32,16 @@ def install
                    "shared_libs=1",
                    "static_libs=0",
                    "install"
+
+    cd "examples" do
+    system "make"
```

Comment:
This needs an indent, or you should be able to do something like `system "make", "-C", "examples"`